### ISSUE TITLE
Tips for restoring a terrible state cluster?

### ISSUE THREAD

**Issue by ns-rlewkowicz** [rook/rook#14064]
I suppose this could be "general support" but there's past documentation issues. Many suggestions revolve around blowing everything out. I have a ton of osds that wont start after traumatic cluster events. 

```
  cluster:
    id:     de239ea9-267a-48b1-992a-37354ce4e9dd
    health: HEALTH_WARN
            1/3 mons down, quorum t,u
            4 osds down
            2 hosts (8 osds) down
            Reduced data availability: 273 pgs inactive
            Degraded data redundancy: 266138/399207 objects degraded (66.667%), 197 pgs degraded, 273 pgs undersized
            3225 daemons have recently crashed

  services:
    mon: 3 daemons, quorum t,u (age 4m), out of quorum: o
    mgr: a(active, since 3m)
    osd: 12 osds: 4 up (since 4m), 8 in (since 13h)
    rgw: 2 daemons active (2 hosts, 1 zones)

  data:
    pools:   11 pools, 273 pgs
    objects: 133.07k objects, 491 GiB
    usage:   485 GiB used, 3.9 TiB / 4.4 TiB avail
    pgs:     100.000% pgs not active
             266138/399207 objects degraded (66.667%)
             197 undersized+degraded+peered
             76  undersized+peered
```

Where do we go on something like this? Knode3 pods just wont start, /var/lib/rook/rook-ceph-system exists. People say blow it out, but I want to keep this?

**Comment by travisn** [comment]
You could try the [troubleshooting guide](https://rook.io/docs/rook/latest/Troubleshooting/ceph-common-issues/). At least the mons still have quorum so you shouldn't need the [disaster recovery guide](https://rook.io/docs/rook/latest/Troubleshooting/disaster-recovery/). 

The rook operator should be attempting to failover the mon and restore full quorum. Do you see any messages in the operator log about if that is failing?

But the core issue is the OSDs. What catastrophe happened? Were disks lost? The key to restoring the cluster will be getting the OSDs back up. What do the crashing OSD logs show?

**Comment by ns-rlewkowicz** [comment]
I had a couple things going. I'll outline steps I took to trouble shoot. 

2 nodes went down. I don't know what went into getting node 2 back, but I think it messed up `/var/lib/rook/`.  

The first node was rejoined as knode01 instead of it's fqdn. However the install is done this is why node one was failing. After resolving this, the osds came back fine (thank goodness). 

So ceph is back at least. The second node is failing with `no disk found with OSD ID 8`. Found tailing the `activate` container. 

```
kubectl logs -f -n rook-ceph-system rook-ceph-osd-8-5c8c9bc6f9-xsxwd -c activate
```

There are 3 missing pvs compared to other nodes so I'm looking there now too.
